site/app/templates/NotificationSettings.twig

```diff
@@ -42,6 +42,44 @@
                 {% endif %}
             </div>
 
+            <div class="heading">
+                <h2>Registration</h2>
+                <hr>
+            </div>
+            <div class="option row">
+                <div class="option-desc col-md-4">
+                    <span>
+                        <span class="option-title">New Self Registration</span>
+                        <br>
+                        <span class="option-alt">Alert me when a student self registers for this course</span>
+                    </span>
+                </div>
+                <div 
+                    class="option-input 
+                    col-md-3 
+                    notification-setting-input">
+                        <input 
+                            type="checkbox" 
+                            aria-label="Notification All New Self Registrations" 
+                            name="self_registration" 
+                            id="self_registration" 
+                            disabled>
+                </div>
+                {% if email_enabled %}
+                    <div 
+                        class="option-input 
+                        col-md-5 
+                        email-setting-input">
+                            <input 
+                                type="checkbox" 
+                                aria-label="Email All New Self Registrations" 
+                                name="self_registration_email" 
+                                id="self_registration_email" 
```

Comment:
As per the [style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/css#styling), IDs and classes should be dash-separated whenever possible.